src/test_l1_hdmi_cec_driver.c

```diff
@@ -245,14 +245,19 @@ void test_hdmicec_hal_l1_open_negative( void )
 
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
     //Check Null even before calling the positive case
+#ifdef ENABLE_ENHANCED_ERROR_CODE
```

Comment:
please update to this requirement -> https://github.com/rdkcentral/ut-core/discussions/55

Reply:
Updated the code as per the suggetion.